core.js

```diff
@@ -16,6 +16,24 @@ const parse5 = require('parse5');
 const fetch = require('window-fetch');
 const {Request, Response, Headers, Blob} = fetch;
 
+const SITE_MODULES = path.join(__dirname, 'site_modules');
+
+const rmrf = path => {
```

Comment:
There are pretty standard modules for doing these.

Reply:
It uses rimraf, which afaik a lot of projects use.